terraform/infrastructure/modules/reporting/ec2.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_instance" "bastion" {
+  count                       = var.bastion_enabled ? 1 : 0
+  ami                         = "ami-00785f4835c6acf64"
+  instance_type               = "t3.micro"
+  subnet_id                   = element(aws_subnet.public_subnet, count.index).id
+  vpc_security_group_ids      = [aws_security_group.bastion.id]
+  associate_public_ip_address = true
+  key_name                    = var.environment == "prod" || length(regexall("ci-", var.environment)) > 0 ? "" : data.aws_key_pair.bastion_key[0].key_name
+  user_data                   = <<EOF
+      #!/bin/bash
+      echo "Installing psql client"
+      sudo yum install postgresql.x86_64
+      echo "Finished installing psql"
+    EOF
+
+  tags = {
+    "Name" = "bastion-${var.environment}"
+  }
+}
+
+
+data "aws_key_pair" "bastion_key" {
```

Comment:
This line is throwing this error when I do a plan. 
<img width="644" alt="image" src="https://github.com/NHSDigital/NRLF/assets/149791/a714374b-9eb5-48f9-b13b-183515d7127f">

Reply:
I've added some instructions to the README for this, you can just set the `bastion_enabled` variable to false which will void the need to create a key pair in EC2. I've also created [NRLF-588](https://nhsd-jira.digital.nhs.uk/browse/NRLF-558) to visit the possibility of using S3 to store the `.pem` file in the future.